pkgs/games/umoria/default.nix

```
@@ -0,0 +1,71 @@
+{ lib, pkgs, gcc9Stdenv, fetchFromGitHub, autoreconfHook, cmake, ncurses6 }:
```

Do not use `pkgs` as a wildcard for packages. List them explicitly.

Also write the argument one per line.

```suggestion
{ lib
, gcc9Stdenv
, fetchFromGitHub
, autoreconfHook
, cmake
, ncurses6
, runtimeShell
}:
```